lib/zapt/commands/runtask.rb

```
@@ -69,6 +83,60 @@ def parse_args string
+          File.join(Dir.pwd, 'cluster_defs', "#{cluster_name}.yaml"),
+          File.join(Dir.pwd, '..', 'common', 'cluster_defs', "#{cluster_name}.yaml")
+        ]
+        yaml_path = possible_paths.find { |p| File.exist?(p) }
+      end
+
+      if yaml_path.nil? || !File.exist?(yaml_path)
+        raise Zapt::Error.new("Can't find cluster definition: #{cluster_name} (tried CloudFormation and YAML)")
+      end
+
+      $logger.info "Loading cluster from YAML: #{yaml_path}" if $logger
+      YAML::load(IO.read(yaml_path))
```

RuboCop (1.81.7)

[convention] Prefer using `YAML.safe_load` over `YAML.load`.

(Security/YAMLLoad)